app/graphql/focus_schema.rb

```diff
@@ -1,4 +1,4 @@
-class GraphqlSchema < GraphQL::Schema
+class FocusSchema < GraphQL::Schema
```

Comment:
Good Question!
The reason I changed the schema is because I thought it would be more explicit to use the schema of the project rather than the general one.